client/templates/users/userEdit.js

```diff
@@ -1,9 +1,56 @@
 Template.userEdit.helpers({
-  //add you helpers here
+  schema: function () {
+    var classSchema = new SimpleSchema({
+      classId: {
+        type: String,
+        allowedValues: function () {
```

Comment:
I think allowedValues can be moved to the non template level schema. This is a validation that should always happen.